### ISSUE TITLE
LoggingConfig for lambda function reported as invalid property

### ISSUE THREAD

**Issue by florian0410** [aws-cloudformation/cfn-lint#2958]
### CloudFormation Lint Version

0.83.3

### What operating system are you using?

Mac

### Describe the bug

In cloudformation template for AWS::Lambda::Function resources, LoggingConfig property is reported as invalid when Cloudformation documentation clearly show it as usable. It's also deploying successfully on Cloudformation with the parameter.

```
E3002 Invalid Property Resources/LambdaFunction/Properties/LoggingConfig
tmp-github-bug.yml:53:7
```

Resources:
https://docs.aws.amazon.com/fr_fr/AWSCloudFormation/latest/UserGuide/aws-resource-lambda-function.html#cfn-lambda-function-loggingconfig
https://docs.aws.amazon.com/fr_fr/AWSCloudFormation/latest/UserGuide/aws-properties-lambda-function-loggingconfig.html

### Expected behavior

No error message should be reported

### Reproduction template

```
AWSTemplateFormatVersion: 2010-09-09
Description: Template for Lambda Sample.
Parameters:
  EnvName:
    Type: String
    Description: Name of an environment. 'dev', 'staging', 'prod' and any name.
    AllowedPattern: ^.*[^0-9]$
    ConstraintDescription: Must end with non-numeric character.

Resources:
  LambdaRole:
    Type: AWS::IAM::Role
    Properties:
      RoleName:
        Fn::Sub: lambda-role
      AssumeRolePolicyDocument:
        Statement:
          - Action:
              - sts:AssumeRole
            Effect: Allow
            Principal:
              Service:
                - lambda.amazonaws.com
        Version: 2012-10-17
      ManagedPolicyArns:
        - arn:aws:iam::aws:policy/AWSLambdaExecute
        - arn:aws:iam::aws:policy/service-role/AWSLambdaBasicExecutionRole
      Path: /

  LambdaFunction:
    Type: AWS::Lambda::Function
    Properties:
      FunctionName:
        Fn::Sub: lambda-function-${EnvName}
      Description: LambdaFunctioni of nodejs16.x
      Runtime: nodejs16.x
      Code:
        ZipFile: |
          exports.handler = function(event, context){
           var sample = sample;
      Handler: ${LambdaHandlerPath}
      MemorySize: 128
      Timeout: 10
      Role:
        Fn::GetAtt:
          - LambdaRole
          - Arn
      Environment:
        Variables:
          ENV:
            Fn::Sub: ${EnvName}
          TZ: UTC
      LoggingConfig:
        LogGroup: !Ref LogGroupTest

  LogGroupTest:
    Type: AWS::Logs::LogGroup
    Properties:
       RetentionInDays: 60
       LogGroupName: "/test/me"

```

**Comment by yamatada-ya** [comment]
aws-sam-translator1.80.0+ supportrs LoggingConfig for lambda function and #2951 may reslove this problem
(I have no idea when this change will be applied, though...)

https://github.com/aws/serverless-application-model/pull/3419/commits

**Comment by kddejong** [comment]
#2951 will include the requirement for 1.80.0

**Comment by kddejong** [comment]
Logging config is in v0.83.4 with the new spec.

**Comment by kddejong** [comment]
You can run `cfn-lint --update-specs` or update the version of cfn-lint to fix this.
